CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Vulnrichment
+
+Thank you for your interest in making the [Vulnrichment](https://github.com/cisagov/vulnrichment) project better!
+Here are some basic guidelines to ensure that we all have the best experience possible with incorporating any
+changes you might be suggesting.
+
+## Opening an issue
+
+In many cases, you may not have a specific fix in mind, and that's okay! Just file an
+[issue](https://github.com/cisagov/vulnrichment/issues), describing the problem you've spotted as best you can,
+ideally with screenshots and any other details you can provide, along with what you expected to happen.
+
+## Pull Requests
+
+Patches accepted! Please note, though, that there's a bit of a wrinkle with submitting PRs against CVE JSON files
+in Vulnrichment repository. Because these files are generated through a backend process here at CISA, your changes
+are quite likely to be overwritten the next time the corpus of enriched CVEs are generated, and your specific
+PR'ed commit may be lost. However, we will make every effort to merge your PR in the normal GitHub way, so even in that
+case, your contribution will be recognized by GitHub and your account will be listed as a contributor. Occasionally,
+though, such merges can create conflicts which we'd rather avoid. This is especially true if you are submitting a PR
+that affects a large number of CVEs. In these cases, the PR is still helpful to see what you're trying to accomplish,
+but the ultimate change may be a systemic change on our backend processing, and your origianl PR may be closed without
```

Comment:
Spelling: original

Reply:
Fixed! Someday I'll figure out the real GitHub workflow of fixing things directly from the PR. I end up always having to open a new window, navigate to the branch (or do it locally on the CLI), and fix it "outside" of the PR. Surely there's a better way.